source/sass/31-molecules/map/_leaflet.scss

```diff
@@ -4,23 +4,24 @@
     border: 0;
 
     a {
-      width: 50px;
-      height: 50px;
-      line-height: 55px;
+      @include box-shadow(0, 3px, 14px, color('gray',-3));
```

Comment:
Je kan ook nog deze parameters als default in de mixin zetten, zodat je overal enkel `@include box-shadow();` moet toevoegen.

Reply:
yes, but i thought the idea was to make it flexible to allow others to use it with whatever shadow they want